velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -406,20 +435,23 @@ std::vector<RowVectorPtr> JoinFuzzer::generateBuildInput(
   // probeInput.
 
   // To ensure there are some matches, sample with replacement 10% of probe join
-  // keys and use these as build keys.
-  // TODO Add a few random rows as well.
+  // keys and use these as 80% of build keys. The rest build keys are randomly
+  // generated.
   std::vector<RowVectorPtr> input;
   for (const auto& probe : probeInput) {
-    auto numRows = 1 + probe->size() / 10;
-    auto build = BaseVector::create<RowVector>(rowType, numRows, probe->pool());
+    auto numRows = 1 + probe->size() / 8;
+    auto build = vectorFuzzer_.fuzzRow(rowType, numRows, false);
 
     // Pick probe side rows to copy.
     std::vector<vector_size_t> rowNumbers(numRows);
+    SelectivityVector rows(numRows, false);
     for (auto i = 0; i < numRows; ++i) {
-      rowNumbers[i] = randInt(0, probe->size() - 1);
+      if (vectorFuzzer_.coinToss(0.8) && probe->size() > 0) {
```

Comment:
Why we need the additional randomness processing here? Does it help catch any new bug? thanks!

Reply:
Hi @xiaoxmeng, yes, when generating a build input row, the original code randomly chooses an existing probe key value as the build key value. Then when the fuzzer tests right join and full join, there is always no unmatched rows from the build side. This PR makes the build input contain existing probe key values for 80% chance, and random build key values for the rest 20% so that there are some unmatched rows in the build input.

Reply:
Maybe comment this in the code. Thanks!